### PR TITLE
55-deleteAComputerBtn

### PR DESCRIPTION
**Pull request by matthamil**
## Status
**READY**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

## Resolves Issue Number

https://github.com/TeamCharles/workforce-management/issues/55

## Deploy Notes

The `project.json` file has been updated. Be sure to run `dotnet restore`.

```sh
git pull --prune
git checkout 55-deleteAComputerBtn
dotnet restore
dotnet run
```

## Steps to Test or Reproduce

1. Navigate to `localhost:5000/Computer`

1. Attempt to delete a computer by clicking the red "X"

1. Read the error message ya goof

1. Attempt to delete a computer that is unassigned.

1. Read the success message, you did it :)

## Impacted Files in Application

* Controllers/ComputerController.cs

* Views/Computer/Index.cshtml

* Startup.cs

* project.json

### CODE REVIEW

**Comment by matthamil** [comment]
@TeamCharles/team Here ya go fam :100:

File: src/workforce-management/Views/Computer/Index.cshtml [hunk 1]
```diff
@@ -2,7 +2,26 @@
 @using Bangazon.Models;
 
 <h2>Computers</h2>
+<<<<<<< HEAD
```
Comment (MattKraatz) [inline src/workforce-management/Views/Computer/Index.cshtml]: Some merge conflict stuff in here

**Review by MattKraatz — APPROVED** [review]
LGTM after a quick change

**Comment by garrett-vangilder** [comment]
LGTM

**Review by Anulfo — COMMENTED** [review]
LGTM

**Comment by daynewright** [comment]
LGTM!

**Pull request merged**